app/models/article.rb

```diff
@@ -153,9 +158,11 @@ def self.unique_url_error
   validates :video_source_url, url: { allow_blank: true, schemes: ["https"] }
   validates :video_state, inclusion: { in: %w[PROGRESSING COMPLETED] }, allow_nil: true
   validates :video_thumbnail_url, url: { allow_blank: true, schemes: %w[https http] }
+  validate :future_or_current_published_at, on: :create
+  validate :has_correct_published_at?, on: :update, unless: :admin_update
 
   validate :canonical_url_must_not_have_spaces
-  validate :past_or_present_date
+  # validate :past_or_present_date
```

Comment:
What are your thoughts on removing commented code? To me commented out code is hard to know if it's removal was intentional or not.